scripts/publish-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+echo "Travis runs in the $(pwd) directory"
+# Set up the user for commiting and pushing.
+git config --global user.email "<mhippely@vmware.com>"
+git config --global user.name "clr-team"
+
+SHA=$(git rev-parse HEAD)
+MESSAGE=$(git log -1 --pretty=format:%s)
+
+# Clone repositories
+git clone https://${GITHUB_OAUTH_BUILD_TOKEN}@github.com/clr-team/clr-angular.git
+git clone https://${GITHUB_OAUTH_BUILD_TOKEN}@github.com/clr-team/clr-icons.git
+git clone https://${GITHUB_OAUTH_BUILD_TOKEN}@github.com/clr-team/clr-ui.git
+
+declare -a repos=("clr-angular" "clr-icons" "clr-ui")
+for repo in "${repos[@]}"
+do
+    # change to the build repo directory
+    cd "./${repo}"
+    # Delete all the files from last build
+    # A filepath to the dist folder in clraity project
+    DIST="$TRAVIS_BUILD_DIR/dist/${repo}/*"
+    # Copy all files into the build repo
+    cp -rf $DIST .
+    # Add any changes to git
+    git add -A
+    # Commit changes with a link to the vmware/clarity commit on master and its message
+    git commit -m "Clarity Build: http://github.com/vmware/clarity/commit/${SHA}\nMessage: ${MESSAGE}"
+    # Push changes up the the build repo
+    git push
+    # Go back up to
+    cd ..
+done
```

Comment:
General question, what about conditions like github is down or there is a problem with cloning/comitting? I suppose this would just error out, but can we get a notification?

Reply:
I can add error handling and notification(s) to slack.
### Error Notifications:
- github down, not able to clone issue
- commiting changes to local repo issue 
- pushing changes to remote repo issue
- dist files didn't build for some reason issue

### Success Notifications
- build success and link to the build repository

I was planning to add a success notification to a slack #clarity-build/#clarity-travis channel but it took me longer than anticipated to iron out running the script on travis. It might take me a little bit longer but I can add it in this PR instead of a different one later.